app/scripts/directives/realtimestring.js

```diff
@@ -165,10 +94,17 @@ angular.module('oncokbApp')
                                 ReviewResource.rollback.push(uuid);
                             }
                         }
+                        // Assign new value to scope.data[scope.key] to update content to gene page when multiple editing at the same time.
+                        if (scope.data[scope.key] !== scope.pureContent.text && !isPropagation) {
+                            scope.data[scope.key] = scope.pureContent.text;
+                        }
                     }
                 }
             },
             controller: function ($scope) {
+                $scope.pureContent = {
```

Comment:
Why bother to create an object with one property only?

Reply:
This is a well-known issue that ng-model cannot monitor the data change. In this case, it doesn't work if you bind to scope.purecontent(string), but it works when you bind to  scope.purecontent.text(object). It's weird but it's true.